src/files.ts

```diff
@@ -129,7 +129,12 @@ export function parseFileIgnores(path: string): string[] {
       );
     }
   }
-  return parseIgnoreRulesToGlobs(rules, dirname);
+  try {
+    return parseIgnoreRulesToGlobs(rules, dirname);
+  } catch (err) {
+    console.error('Could not parse ignore rules to glob', { path, error: (err as Error).message });
+    throw new Error('Ignore rules should be in .gitignore format');
```

Comment:
Can we terminate execution with an error instead of throwing, or is throwing a common approach at the moment?

Reply:
As far as I can see that is the common approach

Reply:
IIRC when a flow throws in the CLI it automatically records error metrics (in Registry), which we might not want in case of invalid ignore files. But also IIRC changing the high-level CLI flow/terminating early is not trivial, so not a blocker if it provides more context and allows users to fix the problem.